src/protocols/rdp/keymaps/fr_ch_qwertz.keymap

```diff
@@ -0,0 +1,45 @@
+parent  "base"
```

Comment:
The license which applies to this file needs to be explicitly stated. See the other keymaps:

https://github.com/jmroth/incubator-guacamole-server/blob/219ce42c3338357dd4fff9dda2944edbdf98af73/src/protocols/rdp/keymaps/de_de_qwertz.keymap#L1-L19